src/kirby/components/avatar/avatar.component.scss

```diff
@@ -1,44 +1,83 @@
 @import "../../scss/web-imports";
+@import '../icon/icon.component.shared';
+
 $image-diameter-small:  40px;
 $image-diameter-medium: 56px;
 $image-diameter-large:  96px;
 
-.image-wrapper {
-  width: $image-diameter-small;
-  height: $image-diameter-small;
-  border-radius: 50%;
-  overflow: hidden;
-  position: relative;
+:host{
+  .avatar {
```

Comment:
I did this because i saw that web styling was overwriting me {N} styling. But i have done the [cba2441](https://github.com/kirbydesign/designsystem/commit/cba244107a90f2ae68ec58929ae2fe854480e51f) solution.